src/pages/Charcuterie.js

```diff
@@ -1,9 +1,12 @@
+import { useState } from 'react'
 import Theme from 'styles/Theme'
 import TextIcon from 'components/TextIcon'
 import Button from 'components/Button'
-import Input from "components/Input";
+import Input from 'components/Input'
+import QuantitySelector from 'components/QuantitySelector'
 
 function Charcuterie({ children }) {
+  const [selectorValue1, setSelectorValue1]= useState(0)
```

Comment:
why are these called xxx1?

Reply:
because there are probably going to be more useState hooks in Charcuterie :) Charcuterie is just draft anyway